### ISSUE TITLE
App reviews indirect research: UI design course

### ISSUE THREAD

**Issue by Arthur-Milchior** [ankidroid/Anki-Android#11829]
TL/DR some students did read 400 reviews and tried to extract useful feedback from them. Exact exercice description is on [assignment 2](https://canvas.sfu.ca/courses/69678/assignments/751350). Their educator selected the 6 best results which are available [here](https://t.co/iVI6rGb34p)

It's from the same lecture as #11664, and I'd love if someone can give it the same treatment. That is, read all of the documents, find what may be actionnable, and create issues from it. The issue should refer the document number and the current issue, so that we can track the origin. In case of doubt, don't hesitate to create an issue. At worst we can close it if we can't use it.

I admit that sometime I'd just love to be able to answer authors to explain why their suggestion is impossible. Or to ask for clarification, because an idea that seems obviously good may be actually hard to define once you try to think about what would be the expected behavior.

Clarification from the educator.
> The [proven] and [unknown] items are to indicate if the student was able to prove or disprove their hypotheses after analyzing the Google App Reviews. They were asked to write their hypotheses before doing the analysis.
